scripts/batchProcessRasters.R

```diff
@@ -9,28 +9,31 @@
 ## The yaml file must be like the example here:
 batchImportRasters_toOSW<-function(inpJSON){
 	
+	#start log timing here...
+	logstart<-format(Sys.time(), "%Y%m%d %H%M%S")
+	
 	## supress warnings
 	options(warn=-1)
 	
 	## libraries an other needes
-	libs<-c("raster","rgdal","sp","airtabler","yaml","jsonlite","aws.s3","remotes")
+	libs<-c("raster","rgdal","sp","airtabler","yaml","jsonlite","aws.s3","remotes","data.table")
 	w<-try(suppressPackageStartupMessages(sapply(libs, require, character.only=T)),silent=T) #check
 	
 	## Are we missing a needed library?
 	if(FALSE %in% w){
 		wdf<-data.frame(Lib=names(w),LoadStatus=as.character(w));row.names(wdf)<-NULL
 		jwdf<-jsonlite::toJSON(subset(wdf,LoadStatus=="FALSE"))
-		retJSON<-paste0("{\"Error\":[{\"Process\":\"Loading libraries\",\"value\":\"Missing libraries\"}], \"Description\":\"",jwdf,"\"}")
+		retJSON<-paste0("{\"Out\":[{\"Result\":\"Error\",\"Process\":\"Loading libraries\",\"value\":\"Missing libraries\"}], \"Description\":\"",jwdf,"\"}")

```

Comment:
Rarely should you need to "hand type" a literal JSON object as a string in any programming or scripting language. It's highly error prone and difficult to read.
Instead, add your key/value pairs to a native object, then convert it to JSON. In an example I found online, a list is created with key/values pairs, then converted to a JSON string. This is the preferred method for all languages, in most situations: https://rdrr.io/cran/rjson/man/toJSON.html

Reply:
Done. I created a function and pass the values of the JSON for it to construct the string